source/fundamentals/builders/projections.txt

```diff
@@ -43,16 +43,64 @@ to any method that expects a projection.
 
    The examples below assume this static import.
 
-The collection is set up with the following documents:
+Sample Documents and Examples
+-----------------------------
```

Comment:
Nit/Suggestion: I think this should be a `~` (one level lower). I do it like this on my pages because I'm introducing the documents before diving into the material, hence why I think it belongs in the Overview and should be one heading size lower opposed to being its own section.